addons/website/static/src/js/content/menu.js

```diff
@@ -622,11 +622,47 @@ publicWidget.registry.hoverableDropdown = animations.Animation.extend({
 
 publicWidget.registry.HeaderMainCollapse = publicWidget.Widget.extend({
     selector: 'header#top',
+    disabledInEditableMode: false,
```

Comment:
@qsm-odoo This change also allowed to fix a bug with the transparency of the collapse menu (with header position = over the content) which was still there in edit mode.